quickstart/pom.xml

```diff
@@ -17,13 +17,10 @@
 
 -->
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <parent>
```

Comment:
Removing the parent makes this module truly independent, while still building it when the parent module is built. IOW, the dependency parent -> quickstart still exists, but the dependency quickstart -> parent is gone. With this, we can be 100% sure that the project can be used as a standalone project stub. The drawback, but we know this already, is that all plugin and dependency versions must be re-declared here.